controllers/JobController.js

```diff
@@ -132,101 +111,56 @@ exports.JobsByEmail = [
  * @returns {Object}
  */
 exports.addJob = [
-	auth,
-	body("title", "Title must not be empty.").isLength({ min: 1 }),
-	body("description", "Description must not be empty.").isLength({ min: 1 }),
-	body("cost", "Cost must be more than $10.").isLength({ min: 1 }).trim(),
-	body("owner", "Owner must not be empty.").isLength({ min: 1 }),
-	body("email", "Email must not be empty.").isLength({ min: 1 }).trim(),
-	body("skills", "Skills must not be less than 3 character.").isLength({ min: 3 }).trim(),
-	sanitizeBody("*").escape(),
-	(req, res) => {
-		try {
-			const errors = validationResult(req);
-			if (!errors.isEmpty()) {
-				return apiResponse.validationErrorWithData(res, "Validation Error.", errors.array());
-			}
-			else {
-				var Job = new JobModel(
-					{
-						title: req.body.title,
-						description: req.body.description,
-						cost: req.body.cost,
-						owner: req.body.owner,
-						email: req.body.email,
-						skills: req.body.skills,
-						date: new Date().toLocaleString('en-US'),
-						status: 'Active',
-						applicants: []
-					});
-				//Save Job.
-				Job.save(function (err) {
-					if (err) { return apiResponse.ErrorResponse(res, err); }
-					let JobData = new jobData(Job);
-					return apiResponse.successResponseWithData(res, "Job add Success.", JobData);
-				});
-			}
-		} catch (err) {
-			//throw error in json response with status 500.
-			return apiResponse.ErrorResponse(res, err);
-		}
-	}
-];
+  auth,
+  body("title", "Title must not be empty.").notEmpty(),
+  body("description", "Description must not be empty.").isLength({ min: 1 }),
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
	body("description", "Description must not be empty.").isLength({ min: 1 }),
```